pkg/reconciler/testing/logger.go

```diff
@@ -28,6 +30,8 @@ func SetupFakeContext(t *testing.T) (context.Context, []controller.Informer) {
 		SendSuccessfully: true,
 	}
 	ctx = cloudevent.WithClient(ctx, &cloudEventClientBehaviour)
+	cacheClient, _ := lru.New(128)
```

Comment:
nit: could you please add an inline comment: `lru.New(/*bufSize=*/ 128)`

Reply:
Oh, in fact this is not needed anymore, it's handled by injection and I forgot to remove it here